startGame.css

```diff
@@ -0,0 +1,6 @@
+#timer{
```

Comment:
id's should not hold style elements only classes

Reply:
done